keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -238,8 +238,8 @@ At a high level, the proposal is to modify the StatefulSet and Job controllers t
 as a pod label at pod creation time (for jobs, this would only apply to jobs in
 Indexed completion mode). The details of this are outlined in the Design Details section below.
 
-StatefulSet pod label: `statefulset.kubernetes.io/pod-index`
```

Comment:
Nit, but would be good to make sure we have this checked out, in the `## Release Signoff Checklist` section make sure to check all the appropriate boxes.